installers/method/kubectl/uninstall.yml

```diff
@@ -0,0 +1,199 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+    name: pgo-install
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+    name: pgo-installer-sa
+    namespace: pgo-install
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+    name: pgo-installer-crb
+    namespace: pgo-install
+roleRef:
+    apiGroup: rbac.authorization.k8s.io
+    kind: ClusterRole
+    name: cluster-admin
+subjects:
+- apiGroup: rbac.authorization.k8s.io
+  kind: User
+  name: system:serviceaccount:pgo-install:pgo-installer-sa
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+    name: pgo-installer
+    namespace: pgo-install
+spec:
+    backoffLimit: 0
+    template:
+        metadata:
+            name: pgo-uninstall
+        spec:
+            serviceAccountName: pgo-installer-sa
+            restartPolicy: Never
+            containers:
+            - name: pgo-uninstaller
+              command: ["/pgo-install.sh", "uninstall"]
+              image: gcr.io/crunchy-dev-test/pgo-installer:centos7-4.3.jmck.7
+              imagePullPolicy: Always
+              env:
+                  - name: HOME
+                    value: "/tmp"
+                  - name: ANSIBLE_CONFIG
+                    value: "/ansible/ansible.cfg"
+                  - name: CRUNCHY_DEBUG
+                    value: "false"
+                  - name: KUBERNETES_IN_CLUSTER
```

Comment:
Do we need every env var set for the uninstall?

Reply:
Yes, these variables are checked in the inventory file each time ansible is run, including uninstall. I think this would correspond with the required environment variables that I pulled from the docs

https://github.com/CrunchyData/postgres-operator/blob/master/installers/ansible/roles/pgo-preflight/tasks/check_inventory.yml